var/spack/repos/builtin/packages/quantum-espresso/package.py

```diff
@@ -600,6 +613,9 @@ def install(self, pkg, spec, prefix):
             else:
                 options.extend(["--with-elpa-lib={0}".format(elpa.libs[0])])
 
+        if "+fox" in spec:
```

Comment:
FYI. In case you haven't seen it, there are helpers from `CMakePackage` for setting options (see https://spack.readthedocs.io/en/latest/build_systems/cmakepackage.html#adding-flags-to-cmake).